.github/ISSUE_TEMPLATE/failing_test.yml

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Failing Test
+description: Report a failing test
+title: "[Failing Test]: "
+labels: ["bug", "awaiting triage", "test-failures", "P1"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report this test failure!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please name which test is failing.
+      placeholder: |
+          If possible include: when the test started failing, a jenkins link, and any initial debugging you've done.
+
+          If this isn't a bug and you have a question or support request, please email user@apache.beam.com with a description of the problem instead of opening this issue.
+  - type: dropdown
+    id: flakey
+    attributes:
+      label: Issue Component
+      options:
+      - "flakey"
+      - "red"
```

Comment:
Could you also update the wording to something like "Test is consistently failing"/"Test is flaky"? A newer contributor might not grok the difference between red and flaky (because a flaky test is sometimes red), **and** that gives us a string to search for in our labeler config.

Second, would you mind adding:

```
- contains: 'Test is flaky' # (or whatever string you decide on)
  addLabels: ['flaky']
```

to the rules in [.github/issue-rules.yml](https://github.com/apache/beam/blob/master/.github/issue-rules.yml)